crates/sui-indexer/src/lib.rs

```diff
@@ -2,16 +2,15 @@
 // SPDX-License-Identifier: Apache-2.0
 #![recursion_limit = "256"]
 
-use std::net::SocketAddr;
-use std::time::Duration;
-
 use anyhow::{anyhow, Result};
 use clap::Parser;
 use jsonrpsee::http_client::{HeaderMap, HeaderValue, HttpClient, HttpClientBuilder};
 use metrics::IndexerMetrics;
 use mysten_metrics::spawn_monitored_task;
 use prometheus::Registry;
+use std::net::SocketAddr;
 use std::path::PathBuf;
+use std::time::Duration;
```

Comment:
I group "use" statements following order of many files in this crate:
std -> other external crates -> sui crates -> this crate

let's keep it this way unless there is some specific reason mergint them?